build/ci/release.yml

```diff
@@ -1468,3 +1519,21 @@ buildvariants:
     run_on: ubuntu1804-small
     tasks:
       - name: .smoke-test .atlascli
+  - name: test_atlascli_repo
+    display_name: "Test AtlasCLI repo"
+    expansions:
+      go_root: "/opt/golang/go1.18"
+      go_bin: "/opt/golang/go1.18/bin"
+    run_on:
+      - ubuntu1804-small
+    tasks:
+      - name: .smoke-test .generate .repo .atlascli
+  - name: test_mongocli_repo
+    display_name: "Test MongoCLI repo"
+    expansions:
+      go_root: "/opt/golang/go1.18"
+      go_bin: "/opt/golang/go1.18/bin"
+    run_on:
+      - ubuntu1804-small
```

Comment:
ubuntu 18.04 will EoL next year, could we run this on 20.04 or 22.04 (22.04 support on evg is currently WIP)?

Reply:
given a bunch of tasks use `ubuntu1804-small` I would be in favour of opening a ticket to move all tasks away from it, does that work for you?

Reply:
works for me